src/citation/schema.py

```diff
@@ -8,6 +8,7 @@
     ARTICLE_JOURNAL,
     BIBTEX_TO_CITATION_TYPES,
     BIBTEX_TYPE_TO_CSL_MAPPING,
+    BLOG_POST,
```

Comment:
Not 100% sure if this is the best type to use for posts? Based off of [CSL types](https://docs.citationstyles.org/en/stable/specification.html#appendix-iii-types), generic `article` seems to make sense.

If not, I think it'd still be good to have [the fields include DOI](https://github.com/ResearchHub/researchhub-backend/blob/fa05231ed7448d475bc0064291548b2e14cad826/src/citation/constants.py#L250) so that posts with DOIs have their DOI show in the metadata section on the reference manager.